exodus/analysis_query/views.py

```diff
@@ -4,29 +4,54 @@
 import random
 import string
 
+from django.conf import settings
 from django.utils.translation import gettext_lazy as _
-from django.forms.models import model_to_dict
 from django.http import HttpResponseRedirect
 from django.http import JsonResponse
 from django.http.response import Http404
 from django.shortcuts import render
 from django.views.generic import FormView, ListView
 
 from exodus.core.apk import StaticAnalysisParameters, start_static_analysis
-from .forms import AnalysisRequestForm
+
+from .forms import AnalysisRequestForm, UploadRequestForm
 from .models import AnalysisRequest
 
 
-def randomword(length):
-    return ''.join(random.choice(string.ascii_lowercase) for i in range(length))
+def random_word(length):
+    return ''.join(random.choice(string.ascii_lowercase) for _ in range(length))
+
+
+def upload_file(request):
+    if request.method == 'POST' and settings.ALLOW_APK_UPLOAD:
+        form = UploadRequestForm(request.POST, request.FILES)
+        if form.is_valid():
+            randhex = str(random_word(60))
```

Comment:
I'm not sure we need this variable

Reply:
Which one?

Reply:
`randhex`
I think we can directly use `req.bucket = str(random_word(60))`, WDYT?